experimental/golden_response_checker_test.go

```diff
@@ -0,0 +1,34 @@
+package experimental
+
+import (
+	"flag"
+	"path/filepath"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+var update = flag.Bool("update", true, "update.golden.data files")
+
+func TestGoldenResponseChecker(t *testing.T) {
+	dr := &backend.DataResponse{}
+
+	//	frame := data.GoldenDF() ????
```

Comment:
would be nice to use the [goldenDF()](https://github.com/grafana/grafana-plugin-sdk-go/blob/master/data/arrow_test.go#L21) function in `test_data`, but I have not figured out how to do the circular references :)